### PR TITLE
Add comments on differences in rasterization between js and native dEQP

### PR DESCRIPTION
**Pull request by Jiawei-Shao**

### CODE REVIEW

File: sdk/tests/deqp/framework/referencerenderer/rrRenderer.js [hunk 1]
```diff
@@ -861,23 +869,32 @@ rrRenderer.triangleRasterizer.prototype.rasterize = function() {
         var barycentric1 = deMath.divide(b1, bSum);
         var barycentric2 = deMath.subtract(deMath.subtract([1, 1, 1, 1], barycentric0), barycentric1);
 
+        // In native dEQP, after rasterization, the pixel (x0, y0) actually represents four pixels:
+        // (x0, y0), (x0 + 1, y0), (x0, y0 + 1) and (x0 + 1, y0 + 1).
```
Comment (zhenyao) [inline sdk/tests/deqp/framework/referencerenderer/rrRenderer.js]: Right, and this caused texture sampling incorrect when texture size is small.

I had a comment somewhere about this, but now I can't find it. :(

**Comment by zhenyao** [comment]
Nice.  Thank you.

File: sdk/tests/deqp/framework/referencerenderer/rrRenderer.js (hunk 1 above)
Comment (Jiawei-Shao) [inline sdk/tests/deqp/framework/referencerenderer/rrRenderer.js]: @zhenyao , do you mean this comment ?
https://github.com/KhronosGroup/WebGL/issues/1690

Comment (zhenyao) [inline sdk/tests/deqp/framework/referencerenderer/rrRenderer.js]: Yes, thank you.

**Pull request merged**